plugins/source/aws/resources/services/cloudwatch/metric_statistics.go

```diff
@@ -0,0 +1,62 @@
+package cloudwatch
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client/tableoptions"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+type statOutput struct {
+	*cloudwatch.GetMetricStatisticsOutput
+	InputJSON tableoptions.CustomCloudwatchGetMetricStatisticsInput `json:"input_json"`
+}
+
+func MetricStatistics() *schema.Table {
+	tableName := "aws_cloudwatch_metric_stats_custom"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html`,
+		Resolver:    fetchCloudwatchMetricStats,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "logs"),
+		Transform:   transformers.TransformWithStruct(&statOutput{}, transformers.WithSkipFields("ResultMetadata"), transformers.WithUnwrapAllEmbeddedStructs()),
```

Comment:
No PKs. Ideal PKs would have to include `input_json` which could complicate things for some dests.

Reply:
I think we should hash the `input_json` (or leave it as a string) so that all destinations can use it as a PK. 

There is a timestamp for each data point `statOutput.Datapoints[].Timestamp` I think that should also be a PK

Reply:
I wouldn't know what to do with those timestamps since they're inside a list (unless you're suggesting each datapoint gets its own row in a sub-table) but I've added `input_hash` to all tables (and `parent_input_hash` to the metric_statistics table, to account for some potential PK clashes)

I couldn't bring myself to encode the json as string just to use it as a string-PK while at the same time making it painful (or inconsistent) to query later.

Reply:
> I do think that each data point should be its own row... That way if a user doesn't have to replace the entire object every single sync

@bbernays giving it a go in https://github.com/cloudquery/cloudquery/pull/11402/commits/ff50cd17ec8560a239b01af2bac425dbad293654